### ISSUE TITLE
Choose columns possibility: Reset to default displayed columns as we do for filters

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#3177]
#### Issue

We have possibility to reset filter for search (clears it) but we haven' possibility to clear custom moved colums of user. How can user reset the columns in any listview to default configuration? Because my employees experimented by this :D and now want default columns on listview... but i sent them in screenshot the default order of them in admin and they must manualy order it.

![cases](https://cloud.githubusercontent.com/assets/5773777/23209862/53e1a732-f8fb-11e6-9a4a-0cfb451d77b1.png)

What is your opinion on this situation ?

#### Your Environment

* SuiteCRM Version used: SuiteCRM Official demo 7.8.1

**Comment by Mausino** [comment]
Those screen which i added was easy... better will it ilustrate this

![case2](https://cloud.githubusercontent.com/assets/5773777/23210248/0abc620c-f8fd-11e6-94fe-14f957156f90.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Choose columns possibility: Reset to default displayed columns as we do for filters](https://trello.com/c/u3I5whEt/23-choose-columns-possibility-reset-to-default-displayed-columns-as-we-do-for-filters)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
